openthread/README.rst

```diff
@@ -8,64 +8,186 @@ This simplifies the OpenThread certification process of the final product.
 
 .. note:: The currently provided libraries are not yet certified.
 
+OpenThread library variants
+***************************
+
+The OpenThread libraries for Nordic devices vary depending on stack configuration and hardware cryptography capabilities.
+User is allowed to choose the stack configuration from one of the predefined values.
+Cryptograpy variant is selected automatically based on the microcontroller capabilities.
 
 Features of the pre-built libraries
 ***********************************
 
 The OpenThread pre-built libraries were created based on commit `656802a76acd2920b74ee4d49363f70aeb3b9621` of OpenThread,
```

Comment:
Commit hash should be updated, right?

Reply:
I don't have the commit yet:) As you mentioned this will be updated after #268